docs/manual/spring-security-docs-manual.gradle

```diff
@@ -62,3 +63,12 @@ def resolvedVersions(Configuration configuration) {
 repositories {
 	maven { url "https://repo.spring.io/release" }
 }
+
+def computeSamplesBranch(String version) {
+	if ('main' == version) {
```

Comment:
I don't think I understand... when could `project.version` be `main`? I would have thought `project.version` would always be a snapshot, milestone, or release version of the artifact.

Reply:
I've made a mistake between version and branch. Gonna fix it